### PR TITLE
Refactor 2015

### PR DESCRIPTION
**Pull request by stephenwright**
Updates to follow some ruby best practices

### CODE REVIEW

File: bin/parse.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+$LOAD_PATH.unshift(root_path) unless $LOAD_PATH.include?(root_path)
+Dir.chdir root_path
+
+def nickParse aliases
+  #File.open 'data/tmp.log', 'w'
+  aliases.each {|a|
+    #File.open 'log' 'r'
+    p "alias: #{a}"
+  }
+#	for n in "${nicks[@]}"; do
```
Comment (tyler-vincent) [inline bin/parse.rb]: Does this commented code need to be around?

Comment (stephenwright) [inline bin/parse.rb]: once this script is functionally equal to parse.sh, i'll make sure there's no commented code

File: lib/known.rb [hunk 2]
```diff
@@ -1,8 +1,7 @@
 #!/usr/bin/env ruby
 
-require_relative('../conf.rb' )
-
-cmd_path = $conf[:dir][:root] + '/cmd'
+root_path = File.expand_path('..', File.dirname(__FILE__))
```
Comment (tyler-vincent) [inline lib/known.rb]: This is being called in each lib often. Would there be any way to store this path somewhere globally and just reference it here?

File: trunkbot/cmd/convo221.rb [hunk 3]
```diff
@@ -1,11 +1,12 @@
...
+
+n1 = ARGV[0]
 n2 = ARGV[1]
 
 begin
 
-  
   raise 'wrong argument count' if ARGV.length != 2
 
   ty = [ 'eight_ender', 'ol_qwerty_bastrd', 'HAM_RADIO', 'tyler' ]
```
Comment (tyler-vincent) [inline trunkbot/cmd/convo221.rb]: In absence of a DB some sort of centralized class for nicknames might not be a bad idea so you don't need to keep defining these

File: trunkbot/cmd/def.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+if ARGV.length < 1
+  puts "usage: def <word> [<number>[-<range>]]"
+
+else
+  ret = WordNet::define ARGV[0] || "test"
+  defs = ret.split("\n")
+
+  if ARGV.length == 2
+  case ARGV[1]
```
Comment (tyler-vincent) [inline trunkbot/cmd/def.rb]: Indenting

**Pull request merged**